test/controllers/foreman/registration_controller_test.rb

```diff
@@ -1,55 +1,51 @@
 require 'katello_test_helper'
 
 class RegistrationControllerTest < ActionController::TestCase
-  describe '#create' do
-    describe 'format_activation_key' do
-      def setup
-        @tax_params = { organization: taxonomies(:organization1).id, location: taxonomies(:location1).id }
-
-        setup_controller_defaults(false)
-        setup_foreman_routes
-        login_user(User.find(users(:admin).id))
-      end
-
-      test do
-        post :create, params: { activation_key: ' key1 ' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1')
-      end
-
-      test do
-        post :create, params: { activation_key: ' key1 , key2 , ' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1%2Ckey2')
-      end
-
-      test do
-        post :create, params: { activation_key: ' , ,,key1, , key2, , ,,,  , ,' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1%2Ckey2')
-      end
-
-      test do
-        post :create, params: { activation_key: ' key1 with space ' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1+with+space')
-      end
-
-      test do
-        post :create, params: { activation_key: ' key1 with space , key2 with space ' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1+with+space%2Ckey2+with+space')
-      end
-
-      test do
-        post :create, params: { activation_key: ' key1 with space , key2 with space ,, ,, , ' }.merge(@tax_params)
-        assert_includes(assigns(:command), 'activation_key=key1+with+space%2Ckey2+with+space')
-      end
-
-      test do
-        post :create, params: { activation_key: '' }.merge(@tax_params)
-        refute_includes(assigns(:command), 'activation_key=')
-      end
-
-      test do
-        post :create, params: @tax_params
-        refute_includes(assigns(:command), 'activation_key=')
-      end
-    end
+  def setup
+    @tax_params = { organization: taxonomies(:organization1).id, location: taxonomies(:location1).id }
+
+    setup_controller_defaults(false)
+    setup_foreman_routes
+    login_user(User.find(users(:admin).id))
+  end
+
+  test do
+    post :create, params: { activation_key: ' key1 ' }.merge(@tax_params)
+    assert_includes(assigns(:command), 'activation_key=key1')
+  end
+
+  test do
```

Comment:
Also please name the tests, for example instead of 
```ruby
test do
...
end
```
instead write
```ruby
def test_activation_key_param_trims_whitespace
...
end
```